### PR TITLE
feat(service): Add docker service ls --filter option

### PR DESCRIPTION
**Pull request by fuentes73**
docker service ls --filter option

### CODE REVIEW

File: python_on_whales/components/service/cli_wrapper.py [hunk 1]
```diff
@@ -353,13 +353,14 @@ def logs(
         else:
             return "".join(x[1].decode() for x in iterator)
 
-    def list(self) -> List[Service]:
+    def list(self, filters: Dict[str, str] = {}) -> List[Service]:
```
Comment (gabrieldemarmiesse) [inline python_on_whales/components/service/cli_wrapper.py:356]: Can you update the docstring with an example? Thanks!

File: tests/python_on_whales/components/test_service.py [hunk 2]
```diff
@@ -45,6 +45,25 @@ def test_get_list_of_services(docker_client: DockerClient):
         assert [my_service] == list_of_services
 
 
+@pytest.mark.usefixtures("swarm_mode")
+def test_filters(docker_client: DockerClient):
```
Comment (gabrieldemarmiesse) [inline tests/python_on_whales/components/test_service.py:49]: Can you try to filter with a label that does not exist and verify that no services are returned? That would ensure that the function is correctly removing services from the result.

**Review by gabrieldemarmiesse — CHANGES_REQUESTED** [review]
Many thanks for the pull, request, this is a cool feature. We just need a few changes and it will good to merge :)

**Comment by fuentes73** [comment]
@gabrieldemarmiesse thanks for your quick feedback. Done, let me know if everything is ok.

**Review by gabrieldemarmiesse — APPROVED** [review]
Many thanks!

**Comment by fuentes73** [comment]
@gabrieldemarmiesse i've had to rebase because it wasn't up to date. Can you enable auto-merge again?

**Pull request merged**